doc/development.rst

```diff
@@ -44,11 +44,11 @@ vagrant box will attempt to mount them over SSH.
 
 
 To run the vagrant development environment make sure you have a support
-virtualization system as well as vagrant installed and execute ``vagrant up``.
+virtualization system as well as vagrant installed and execute ``./tools/vagrantup``.
 
 .. warning::
 
-   The initial run updates the systems and can take some time. To provision faster try ``vagrant up --parallel servers fedora-cloud fedora-atomic && vagrant up commissaire``.
+   If you want to use the ``vagrant`` command directly note that you will have to follow the same start up process used in ``./tools/vagrantup``
 
```

Comment:
In the note below that describes the manual work-around for `fedora-atomic`, can you tack on:
"(The `./tools/vagrantup` script does this automatically.)"